addons/project/static/tests/project_task_list_view.test.js

```diff
@@ -0,0 +1,47 @@
+import { describe, expect, test } from "@odoo/hoot";
+import { check, queryAll, queryOne } from "@odoo/hoot-dom";
+import { animationFrame } from "@odoo/hoot-mock";
+import { mountView } from "@web/../tests/web_test_helpers";
+
+import { defineProjectModels, ProjectTask } from "./project_models";
+
+
+defineProjectModels();
+
+describe.current.tags("desktop");
```

Comment:
@damrOdoo I don't know but I guess it is to be able to launch those tests in desktop only? 🤷🏼‍♂️

Reply:
@damrOdoo @xavierbol Yes, with this tag, it launches the test in desktop only (not in mobile). For example, a desktop view can have three visible buttons, whereas mobile have a dropdown containing those three buttons. If your test checks that there are 3 buttons, it will obviously crash in mobile, so you write this line for the test to be run only on desktop :ok_hand:

Reply:
Thanks for the explanation! 🙂